var/spack/repos/builtin/packages/deconwolf/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Deconwolf(CMakePackage):
+    """deconwolf is a software for 3-D deconvolution of fluorescent wide-field images."""
+
+    homepage = "https://deconwolf.fht.org/"
+    git = "https://github.com/elgw/deconwolf.git"
+    url = "https://github.com/elgw/deconwolf/archive/refs/tags/v0.4.5.tar.gz"
+
+    maintainers("dacolombo")
+
+    license("GPL-3.0-only")
+
+    version("0.4.5", tag="v0.4.5")
```

Comment:
```suggestion
    version("0.4.5", tag="v0.4.5", commit="ca062c49eedd6bf60fcd9804df467e64cfc3c113")
```

Since tags are floating versions, we should always lock them down to specific commits for reproducibility.

Or you can have the version come from the automatically generated tarball that you linked in the URL.

```
version("0.4.5", sha256="e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855")
```